tools/azure-sdk-tools/devtools_testutils/powershell_preparer.py

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import functools
+import os
+
+from . import AzureMgmtPreparer
+from .resource_testcase import RESOURCE_GROUP_PARAM
+from azure_devtools.scenario_tests.exceptions import AzureTestError
+
+class PowerShellPreparer(AzureMgmtPreparer):
```

Comment:
Can you help me understand what this script is doing?

Reply:
This is a python fixture that delivers the environment variables to a test and properly scrubs test secrets from the recordings. For an example of how it's used, check [here](https://github.com/Azure/azure-sdk-for-python/blob/1fabc1fc67c4fe9b9fed01293d58e75947d21f94/sdk/tables/azure-data-tables/tests/test_table_client_cosmos.py#L25-L28) for instantiation and [here](https://github.com/Azure/azure-sdk-for-python/blob/1fabc1fc67c4fe9b9fed01293d58e75947d21f94/sdk/tables/azure-data-tables/tests/test_table_client_cosmos.py#L58-L71) for inline use.